config/main.py

```diff
@@ -189,6 +189,94 @@ def load_minigraph():
     _restart_services()
     print "Please note setting loaded from minigraph will be lost after system reboot. To preserve setting, run `config save`."
 
+#
+# 'vlan' group
+#
+@cli.group()
+@click.pass_context
+@click.option('-s', '--redis-unix-socket-path', help='unix socket path for redis connection')
+def vlan(ctx, redis_unix_socket_path):
+    """VLAN-related configuration tasks"""
+    kwargs = {}
+    if redis_unix_socket_path:
+        kwargs['unix_socket_path'] = redis_unix_socket_path
+    config_db = ConfigDBConnector(**kwargs)
+    config_db.connect(wait_for_init=False)
+    ctx.obj = {'db': config_db}
+    pass
+
+@vlan.command('add')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.pass_context
+def add_vlan(ctx, vid):
+    db = ctx.obj['db']
+    vlan = 'Vlan{}'.format(vid)
+    if len(db.get_entry('VLAN', vlan)) != 0:
+        print "{} already exists".format(vlan)
+        raise click.Abort
+    db.set_entry('VLAN', vlan, {'vlanid': vid})
+
+@vlan.command('del')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.pass_context
+def del_vlan(ctx, vid):
+    db = ctx.obj['db']
+    db.set_entry('VLAN', 'Vlan{}'.format(vid), None)
```

Comment:
Would it be better to delete vlan members before deleting the vlan itself?

Reply:
Thanks for your review. fixed.